File: AngularAutoSaveCommands/angularApp/app/services/CommandDto.ts
export class CommandDto {

    public commandType: string;
    public payloadType: string;
    public payload: any;
    public actualClientRoute: string;

    constructor(commandType: string, payloadType: string, payload: any, actualClientRoute: string) {

        this.commandType = commandType;
        this.payloadType = payloadType;
        this.payload = payload;
        this.actualClientRoute = actualClientRoute;

    }
}

File: AngularAutoSaveCommands/Models/CommandDto.cs
using Newtonsoft.Json.Linq;
using System;
using System.Text.Json;

namespace AngularAutoSaveCommands.Models
{
    public class CommandDto
    {
        public string CommandType { get; set; }

        public string PayloadType { get; set; }

        public JObject Payload { get; set; }

        public string ActualClientRoute { get; set;}

    }

    public class CommandDtoObject
    {
        public string CommandType { get; set; }

        public string PayloadType { get; set; }

        public Object Payload { get; set; }

        public string ActualClientRoute { get; set; }

    }
}


